src/exporto0o.ts

```diff
@@ -178,7 +179,8 @@ export async function exportToOo(
 
   try {
     console.log(`[${plugin.manifest.name}]: export command: ${cmd}`);
-    await exec(cmd, { cwd: variables.currentDir, env });
+    // It is necessary to **append** to the current TEXINPUTS - NOT REPLACE. TEXINPUTS contains the basic latex classes. 
+    await exec(cmd, { cwd: variables.currentDir, env: { TEXINPUTS: joinEnvPath(textemplateDir, process.env.TEXINPUTS) } }); 
```

Comment:
These changes are unnecessary(The `env` is important for https://github.com/mokeyish/obsidian-enhancing-export/pull/83). 


You can preset the `TEXINPUTS` here:

https://github.com/mokeyish/obsidian-enhancing-export/blob/054b98469bf64df76a6628570b8fd09dfe9c2936/src/settings.ts#L86

See here: https://github.com/mokeyish/obsidian-enhancing-export/pull/84#issuecomment-1594797523

Reply:
It should be :

```js
{
   'PATH': '/usr/local/bin:/Library/TeX/texbin:${PATH}',
   'TEXINPUTS': '${pluginDir}/textemplate:${TEXINPUTS}'
}

```

Reply:
Neat, this is very elegant.

Reply:
@mokeyish Sorry I missed that. Should be fixed now.